### PR TITLE
Task/receive image/message image class

### PR DESCRIPTION
**Pull request by teamagam1**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/teamagam/GimelGimel/pull/18" target="_blank">ReviewNinja

File: app/src/main/java/com/teamagam/gimelgimel/app/model/entities/ImageMetadata.java [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+ * MessageImage Content.
+ */
+public class ImageMetadata {
+
+
+    @Retention(RetentionPolicy.SOURCE)
+    @StringDef({USER, Sensor})
+    public @interface SourceType {}
+    public static final String USER = "User";
+    public static final String Sensor = "Sensor";
```
Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/model/entities/ImageMetadata.java]: uppercase that !fix

File: app/src/main/res/values/config.xml [hunk 2]
```diff
@@ -7,7 +7,4 @@
     <integer name="location_min_update_frequency_ms">1000</integer>
     <string name="location_threshold_update_distance_m">3</string>
 
-    <string name="GG_api_base_url">http://ggmessaging.herokuapp.com/</string>
```
Comment (teamagam3) [inline app/src/main/res/values/config.xml:10]: Should it be here?
Consider revert this file

File: app/src/main/java/com/teamagam/gimelgimel/app/view/MainActivity.java [hunk 3]
```diff
@@ -118,10 +120,19 @@ public void run() {
                 });
             }
         };
+
+
+
         mTextMessageReceiver = new MessageBroadcastReceiver(messageHandler, Message.TEXT);
         mLatLongMessageReceiver = new MessageBroadcastReceiver(messageHandler, Message.LAT_LONG);
+        mImageMessageReceiver = new MessageBroadcastReceiver(new MessageBroadcastReceiver.NewMessageHandler() {
```
Comment (teamagam3) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/MainActivity.java:128]: If it's for testing only, you can remove it

File: app/src/main/java/com/teamagam/gimelgimel/app/model/entities/ImageMetadata.java (hunk 1 above)
Comment (teamagam1) [inline app/src/main/java/com/teamagam/gimelgimel/app/model/entities/ImageMetadata.java]: !fixed

Comment (teamagam1) [inline app/src/main/java/com/teamagam/gimelgimel/app/model/entities/ImageMetadata.java]: !fixed

File: app/src/main/java/com/teamagam/gimelgimel/app/view/MainActivity.java [hunk 4]
```diff
@@ -118,10 +120,19 @@ public void run() {
...
+            @Override
+            public void onNewMessage(Message msg) {
+                Toast.makeText(MainActivity.this, msg.getContent().toString(), Toast.LENGTH_LONG).show();
+            }
+        }, Message.IMAGE);
```
Comment (teamagam1) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/MainActivity.java:133]: this is place holder for the next task. !completed

File: app/src/main/res/values/config.xml (hunk 2 above)
Comment (teamagam1) [inline app/src/main/res/values/config.xml:10]: I'm not sure where it will be. now it is just confusing

**Pull request merged**